src/ui/templates/insurance/your-buyer/your-buyer-details.njk

```diff
@@ -0,0 +1,91 @@
+{% extends 'index.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+{% from 'govuk/components/label/macro.njk' import govukLabel %}
+{% from "govuk/components/hint/macro.njk" import govukHint %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from "govuk/components/error-message/macro.njk" import govukErrorMessage %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+
+{% block pageTitle %}
+  {{ CONTENT_STRINGS.PAGE_TITLE }}
+{% endblock %}
+
+{% block content %}
+
+  {{ govukBackLink({
+    text: CONTENT_STRINGS.LINKS.BACK,
+    href: BACK_LINK,
+    attributes: {
+      "data-cy": "back-link"
+    }
+  }) }}
+
+  {% if validationErrors.summary %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: validationErrors.summary
+    }) }}
+  {% endif %}
+
+  <form method="POST" novalidate>
+
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+    <input type="hidden" name="{{ HIDDEN_FIELD_ID }}" id="{{ HIDDEN_FIELD_ID }}" value="{{ submittedValues[HIDDEN_FIELD_ID].name }}">   
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half-from-desktop">
+
+        {% set class = "govuk-form-group" %}
+
+        {% if validationErrors.errorList[FIELD_ID] %}
+          {% set class = "govuk-form-group govuk-form-group--error" %}
+        {% endif %}
+
+
+{{ govukSelect({
+  id: CONTENT_STRINGS.BUYER_COUNTRY,
+  name: CONTENT_STRINGS.BUYER_COUNTRY,
+  label: {
+    text: CONTENT_STRINGS.COUNTRY_LABEL
+  },
+  items: countries
+}) }}
+      </div>
+
+            <div class="govuk-grid-column-three-quarters-from-desktop">
+        <div class="govuk-button-group">
+          {{ govukButton({
+            text: CONTENT_STRINGS.BUTTONS.CONTINUE,
+            attributes: {
+              'data-cy': 'submit-button'
+            }
+          }) }}
+
+          {{ govukButton({
+            text: CONTENT_STRINGS.BUTTONS.SAVE_AND_BACK,
+            classes: "govuk-button--secondary",
+            attributes: {
+              'data-cy': 'save-and-back-button'
+            }
+          }) }}
+        </div>
+      </div>
+
+    </div>
+
+
+
+  </form>
+
+  <script src="/assets/js/accessibleAutocomplete.js" type="text/javascript"></script>
+
+  <script type="text/javascript">
+    var element = document.getElementById('{{ FIELD_ID }}');
+    accessibleAutocomplete.enhanceSelectElement({
+      selectElement: element,
+      defaultValue: ''
+    });
+  </script>
```

Comment:
No need for the scripts, this is for eligibility buyer country which is different requirement